test/README.md

```diff
@@ -126,3 +126,29 @@ You should see the test pass, checking that message delivery works correctly and
 ## Generating/Updating new test fixtures
 
 Test fixtures are taken by running the service in full e2e test. The relayer should log the fixture data you need (code is in [the relayer here](../relayer/workers/beefyrelayer/fixture-data-logger.go), though may require a bit of manual copy/pasting to get perfectly it in the right format.
+
+## Running E2E tests on Ropsten
+
+To run the E2E tests on Ropsten you need to have separate accounts for the relayers, an account for deployment and one for running the E2E test stack. You will also require an [Infura](https://infura.io/) account and project.
+
+Add the following variables to `ethereum/.envrc`:
```

Comment:
I don't the think the variables from `ethereum/.envrc` are used at all when starting the E2E stack? Can probably remove this part if thats the case.